deployments/btcstaking-bitcoind/btcstaking-demo.sh

```diff
@@ -94,20 +150,31 @@ docker exec finality-provider0 /bin/sh -c \
 echo "Finality Provider with Bitcoin public key $attackerBtcPk submitted a conflicting finality signature for Babylon height $attackHeight; the Finality Provider's private BTC key has been extracted and the Finality Provider will now be slashed"
 
 echo "Wait a few minutes for the last, shortest BTC delegation (10 BTC blocks) to expire..."
-sleep 10
+sleep 100
```

Comment:
lets create another test for multisig support and not modify the btcstaking-demo